src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String insertDriverRequest = "INSERT INTO drivers(name, license_number) "
+                + "VALUES(?, ?);";
```

Comment:
```suggestion
        String insertDriverRequest = "INSERT INTO drivers (name, license_number) "
                + "VALUES (?, ?);";
```

Reply:
I added spaces in this places, made commit, but here nothing changed, don't understand why

Reply:
I added spaces in this places, made commit, but here nothing changed, don't understand

Reply:
It did)